File: MDR.Web.UI/app/contact/contactController.js
(function () {
    "use strict";
    angular.module("mdrApp").
    controller("contactController", ContactController);

    function ContactController() {
        var vm = this;
        vm.sendMessage = function (isValid) {
            if (isValid) {
                alert("Form is valid");
            }
            else {
                alert("Form is invalid");
            }
        };
    };
})();

File: MDR.Web.UI/Controllers/ContactController.cs
using MDR.Web.UI.Models;
using System.Web.Mvc;

namespace MDR.Web.UI.Controllers
{
    public class ContactController : Controller
    {
        public ActionResult Index()
        {
            var message = new ContactMessageViewModel();
            return View(message);
        }

        [HttpPost]
        public JsonResult SendMessage(ContactMessageViewModel message)
        {
            if (ModelState.IsValid)
            {
                // Process form submit
            }
            return Json("Error");
        }
    }
}